pydantic_models/pharmacogenomics.py

```diff
@@ -55,6 +55,9 @@ class VariantAnnotation(BaseModel):
     entity: Optional[str] = Field(
         description="Entity affected by the effect.", examples=["malignant hyperthermia"]
     )
+    id: Optional[str] = Field(
```

Comment:
Did you consider `studyId`? It's also available in the root, and it represents PharmGKB's ID for the clinical annotation

Reply:
It's not really a study id, this is the identifier of the annotation. I know, we did some bending of schemas for evidence, but in the case of pharmacogenetics, it is only one PharmGKB, and I'm not sure when would be an other source, where we would need to consolidate the labels. Also, we decided not to use `annotationId`, because the object is already called `annotation`. I'm not sure if this has addressed your question.